### DISCUSSION TITLE
Zero-Knowledge?

### DISCUSSION THREAD

**Discussion by jo-elimu** [jo-elimu/ethtaipei2024-hackathon#3] [General]
Should we include zero-knowledge (zk) functionality? Is it necessary to preserve the privacy of a DAO member that rated another DAO member?

**Comment by juroberttyb** [comment]
I guess this would be a really nice feature, but maybe add it after we have a functional prototype of the dao implementation?

**Reply by juroberttyb** [reply]
Not sure if it would be hard to implement.

**Comment by dalaoqi** [comment]
I think it's okay if it's just about ratings. But if there are reviews and personal info involved, it might be necessary?

**Comment by jo-elimu** [comment]
@juroberttyb @dalaoqi Yes, sounds good. Let's just build the ratings functionality first, and if we have any time left, we can experiment with adding privacy (zero-knowledge) features 👍🏽
